meqa/mqplan/dsl.go

```diff
@@ -1399,6 +1414,7 @@ func RandomTime(t time.Time, r time.Duration) time.Time {
 // date ranges. Prefix is a prefix to use when generating strings. It's only used when there is
 // no specified pattern in the swagger.json
 func generateString(s mqswag.SchemaRef, prefix string) (string, error) {
+	s.Value.Pattern = generatePattern(s.Value.Format)
```

Comment:
For email, we have set the value again `s.Value.Pattern = "^[a-z0-9]+@[a-z_]+?\\.[a-z]{2,3}$"` Is that expected?

Reply:
Nope, thanks for pointing out.
Removed it.